docs/docs/howto/workload-deployment/generate-annotations.md

```diff
@@ -114,21 +114,15 @@ This must be done on a trusted machine, with a secure and trusted connection to
 contrast generate --reference-values k3s-qemu-tdx resources/
 ```
 
-On bare-metal TDX, `contrast generate` is unable to fill in the `MinimumTeeTcbSvn` and `MrSeam` TCB values as they can vary between platforms.
-They will have to be filled in manually.
+On bare-metal TDX, `contrast generate` is unable to fill in the `MrSeam` value as it depends on your platform configuration.
```

Comment:
MinimumTeeTcbSvn is also mentioned on `getting-started/deployment.md` and `howto/encrypted-storage.md`

Reply:
I know, there is some duplication we have to remove. The previous PR also didn't touch those. Will fix in a follow-up.